krate/src/main/kotlin/hu/autsoft/krate/Functions.kt

```diff
@@ -46,6 +47,10 @@ public fun Krate.longPref(key: String): ReadWriteProperty<Krate, Long?> {
 
 /**
  * Creates an optional preference of type String with the given [key] in this [Krate] instance.
+ *
+ * @param [key] the key to identify this value in SharedPreferences
+ *
+ * @return [StringDelegate]
```

Comment:
In general, I don't think `@param` and `@return` parts are required in these comments. The one-liner comment already described what the function does, and referenced the parameter as well, with `[key]`.

Also, the value being stored in `SharedPreferences` is sort of an implementation detail, so the docs don't have to reference it, and so is the concrete type (`StringDelegate`) of the object returned by this function.

Reply:
This change to the comment on the original `stringPref` function should still be reverted.